prometheus_hardware_exporter/collectors/ipmi_sel.py

```diff
@@ -23,7 +23,9 @@ def get_sel_entries(self, time_range: int) -> Optional[List[Dict[str, str]]]:
         Returns:
             sel_entries: a list of dictionaries containing sel_sentries, or []
         """
-        result = self("--output-event-state --interpret-oem-data --entity-sensor-names")
+        result = self(
+            "--sdr-cache-recreate --output-event-state --interpret-oem-data --entity-sensor-names"
```

Comment:
**nitpick (blocking):** 

Can you add some comments the mention why those options is required and also add unit-test to it?

Reply:
Thanks, I have added comments on why the flag is needed. 
For the unit-test, however, [we already have tests that mock __call__](https://github.com/canonical/prometheus-hardware-exporter/blob/6c55a2d4665480204060bfcd09512e702f0e7c45/tests/unit/test_ipmi_sel.py#L49), which mock this call to `ipmi-sel` and `ipmimonitoring` in this case, so the flags don't really matter.